doc/tutorial/Server.lhs

```diff
@@ -1020,6 +1020,16 @@ serverFor = error "..."
 -- or the mailing list if you get stuck!
 ```
 
+If the API contains the `EmptyAPI` combinator, the corresponding server is
+called `emptyServer`:
```

Comment:
Could you describe the behavior of `emptyServer` in a few words? Maybe a better wording overall would be:

> When your API contains the `EmptyAPI` combinator, you'll want to use `emptyServer` in the corresponding slot for your server, which will simply fail with 404 whenever a request reaches it.

?